### PR TITLE
Bootstrap the repo

### PR DESCRIPTION
**Pull request by amwat**
k8s.io/test-infra/kubetest2 -> sigs.k8s.io/kubetest2

/cc @BenTheElder

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module sigs.k8s.io/kubetest2
+
+go 1.14
+
+require (
+	github.com/aws/aws-k8s-tester v1.0.0
```
Comment (BenTheElder) [inline go.mod:6]: we should get this into a sub-module (just containing a tester?) for sure.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.PHONY: install
+install:
+	go install .
```
Comment (BenTheElder) [inline Makefile:3]: force module mode?

Comment (amwat) [reply Makefile]: done

File: go.mod (hunk 1 above)
Comment (amwat) [reply go.mod:6]: you mean get a different go.mod for the eks deployer right?
i will probably do that later with the correct naming sorted out.

**Comment by cofyc** [comment]
/lgtm

ready to merge?

**Pull request merged**